torch/csrc/Size.cpp

```diff
@@ -41,7 +41,7 @@ PyObject * THPSize_NewFromSizes(int dim, const int64_t *sizes)
 static bool isTracedVar(PyObject *item) {
   if (!THPVariable_Check(item)) return false;
   auto & var = reinterpret_cast<THPVariable*>(item)->cdata;
-  return torch::jit::tracer::isTracing(var);
+  return var.dim() == 0 && torch::jit::tracer::getValueTrace(var);
```

Comment:
dim() == 0?

Reply:
Yeah, we want to allow putting tensors into `torch.Size`s, but they need to be scalars